### ISSUE TITLE
Add aria-labels to contenteditable items

### ISSUE THREAD

**Issue by enatario** [russellgoldenberg/responsive-table-generator#30]
Apparently screen readers have issues reading content within a `contenteditable` item. So if you add a `aria-label='your label'` attribute to the html element, you should be good to go. Now this is just nitpicking and making the generator site and not the actual table accessibility-compliant, but whatevs.
